chrome/content/zotero/preferences/preferences_general.xul

```diff
@@ -32,6 +32,7 @@
 		
 		<preferences id="zotero-prefpane-general-preferences">
 			<preference id="pref-fontSize" name="extensions.zotero.fontSize" type="string"/>
+			<preference id="pref-layout" name="extensions.zotero.layout" type="string" instantApply="true" onchange="Zotero.getActiveZoteroPane().updateLayout();"/>
```

Comment:
(I like instantApply on OS X, but none of the other prefs on Windows and Linux do it.)